src/app/shared/directives/tooltip.directive.ts

```diff
@@ -0,0 +1,70 @@
+import { ApplicationRef, ComponentRef, createComponent, Directive, EventEmitter, HostListener, Input, OnDestroy, Output, TemplateRef } from '@angular/core'
+import { TooltipComponent } from '../components/tooltip/tooltip.component'
+
+const MINIMUM_HORIZONTAL_OFFSET_IN_PIXELS: number = 0
+
+export interface TooltipMetadata {
+  horizontalOffsetInPixels: number
+  hostElementWidth?: number
+}
+
+@Directive({
+  selector: '[sofieTooltip]',
+})
+export class TooltipDirective implements OnDestroy {
+  @Input() public sofieTooltipTemplate: TemplateRef<unknown>
+
+  @Output() public tooltipMetadataChange: EventEmitter<TooltipMetadata> = new EventEmitter()
+
+  private tooltipComponentRef: ComponentRef<TooltipComponent>
+
+  constructor(private readonly applicationRef: ApplicationRef) {}
+
+  @HostListener('mouseenter', ['$event'])
+  public showTooltip(event: MouseEvent): void {
+    this.createTooltip()
+    this.moveTooltip(event)
+  }
+
+  private createTooltip(): void {
+    this.tooltipComponentRef = createComponent(TooltipComponent, {
+      environmentInjector: this.applicationRef.injector,
+    })
+
+    this.tooltipComponentRef.instance.templateRef = this.sofieTooltipTemplate
+    this.tooltipComponentRef.changeDetectorRef.detectChanges()
+
+    document.body.append(this.tooltipComponentRef.location.nativeElement)
+  }
+
+  @HostListener('mousemove', ['$event'])
+  public moveTooltip(event: MouseEvent): void {
+    if (!this.tooltipComponentRef) {
+      return
+    }
+
+    const verticalPositionInPixels: number = event.clientY - event.offsetY
```

Comment:
If we compute the bounding rectangle of the element, and use the top value, then the jitter is removed:
```typescript
const triggerElement: Element | undefined = event.target as Element | undefined
const verticalPositionInPixels: number = triggerElement?.getBoundingClientRect().top ?? 0

Reply:
Updated